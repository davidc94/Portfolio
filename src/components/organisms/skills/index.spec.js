import React from 'react';
import renderer from 'react-test-renderer';
import Skills from './index';

describe('Skills component', () => {
  it('render Skills as expected', () => {
    const render = renderer.create(
      <Skills />
    );
    expect(render.toJSON()).toMatchSnapshot();
  });
});