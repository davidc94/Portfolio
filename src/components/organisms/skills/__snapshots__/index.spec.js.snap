// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skills component render Skills as expected 1`] = `
<div
  className="sc-bwzfXH bejfXd"
  width="full"
>
  <p
    className="sc-bdVaJa kAfGma"
    color="#EEE5E9"
    size="large"
  >
    Skills
  </p>
  <p
    className="sc-bdVaJa kRyTSY"
    size="base"
  >
    Here are some of the skills ðŸ¥‹ I've picked up along the way
  </p>
  <div
    className="sc-bwzfXH hknQqq"
  >
    <li
      className="sc-htpNat eAyckY"
      size="base"
    >
      React Native
    </li>
    <li
      className="sc-htpNat eAyckY"
      size="base"
    >
      React
    </li>
    <li
      className="sc-htpNat eAyckY"
      size="base"
    >
      Jest
    </li>
    <li
      className="sc-htpNat eAyckY"
      size="base"
    >
      GraphQL
    </li>
    <li
      className="sc-htpNat eAyckY"
      size="base"
    >
      App Development
    </li>
    <li
      className="sc-htpNat eAyckY"
      size="base"
    >
      CI/CD Pipelines
    </li>
    <li
      className="sc-htpNat eAyckY"
      size="base"
    >
      Jenkins
    </li>
  </div>
</div>
`;
